protobuf-c.spec

```diff
@@ -1,19 +1,39 @@
 Name:           protobuf-c
-Version:        1.3.0
+Version:        1.3.1
 Release:        1%{?dist}
 Summary:        C bindings for Google's Protocol Buffers
 
-
+%if 0%{?suse_version} >= 1315
+Group:          Development/Libraries/C and C++
+License:        BSD-2-Clause
+%else
 Group:          System Environment/Libraries
 License:        BSD
+%endif
+
 URL:            https://github.com/protobuf-c/protobuf-c
 Source0:        https://github.com/protobuf-c/protobuf-c/releases/download/v%{version}/%{name}-%{version}.tar.gz
 
+%if 0%{?suse_version} >= 1315
```

Comment:
Why is this SLES only?  It's probably applicable to all platforms.

Reply:
That is the package name that SLES uses.  Red Hat uses "pkgconfig" and seems to already include it in the chroot.

Reply:
Even if `mock` does implicitly include it in the chroot, if the package wants it to build, it should be specified as a `BuildRequires:`.

Reply:
Ok.  We probably have a few other spec files that will eventually need that change.

Reply:
Looks like RedHat has modfied rpm packaging in fedora to use pkgconfig(package-name) notation in their BuildDepends where it uses pkgconfig to identify what the packages need.
https://fedoraproject.org/wiki/PackagingDrafts/PkgConfigBuildRequires